todo_app/data/test_view_model.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from todo_app.data.trello_items import Item
+from todo_app.data.view_model import ViewModel
+
+def test_done_items_property():
```

Comment:
Interesting to note that the fact this test passes hasn't helped us catch the bug mentioned above - and this is because the bug is really about how the Item class is put together by the `fetch_todo_items` function - which we don't test in isolation - and because we never use these functions that your unit testing in our actual code so there's no issue in the integration test

So while these are good tests, if this was a real code base we'd want to throw them out because they're testing something we never use! Here instead I'd recommend that we update the code so that we _are_ using them 🙂